visiting.md

```diff
@@ -1,20 +1,60 @@
 
-### Are the venues accessible?
+### Community billets
 
-We are committed to reducing barriers to access throughout the event. All spaces—for Kickoff Yami-ichi, Conference, and Sprints—have elevator access and accessible washrooms.
+Billeting is a way of supporting participants to attend within their budget and at the same time, welcome them into our community. 
+
+We anticipate securing a limited number of spots with people based in Toronto. If you indicated you were interested in hosting or staying in a community billet check out the [community billets list](https://ethercalc.org/s0r2i3dsfznl) and add your info or email billets@ournetworks.ca and we’ll help coordinate!
+
+### Small-businesses and BIPOC-owned hotels
+
+- [Baldwin Village Inn](https://www.urbannorthinns.com/baldwin-village-inn/) and [Urban North Inns](https://www.urbannorthinns.com/), 9 Baldwin Street
+
+We searched and could not confirm a directory of BIPOC-owned accomodations, if you know of any, please contact us. 
+
+### Unionized hotels
+
+We used [Fair Hotels](https://www.fairhotel.org/) and called []
+Called [UNITE HERE 75](https://www.uniteherelocal75.org/) for leads and updates, As of May 2019, these hotels are still unionized with them:
+
+- [Chelsea Hotel Toronto](http://www.chelseatoronto.com/en/), 33 Gerrard Street West
+- [Kimpton Saint George Hotel](https://www.ihg.com/kimptonhotels/hotels/us/en/saint-george-hotel-toronto-on/yyzbs/hoteldetail), 280 Bloor Street West
+- [One King West Hotel & Residence](https://www.onekingwest.com/)
+- [The Omni King Edward Hotel](https://www.guestreservations.com/the-omni-king-edward-hotel/booking)
+- [Marriott St. Regis](https://www.marriott.com/hotels/travel/yyzxr-the-st-regis-toronto/)
+
+### Other Short-term rentals
+
+Additional care to take if engaging in a short-term rental in Toronto:
+
+1. Avoid hosts with multiple listings.
+1. Avoid AirBnB rentals in the following neighbourhoods:
+  - Parkdale
+1. Avoid these hosts identified by a local nonprofit as AirBnB "ghost hotel" landlords.
+    
+Please feel free to use this text for your conference in Toronto.
+
+---
+
+Would you like to make a guide like this for your event or city? If so:
+
+- Search for reports from local housing justice organizations. Read them carefully, summarize their insights, and link to them. Use their recommendations to craft your suggestions.
+- Categories of hotels to look for include: unionized hotels, small businesses, indigenous-owned hotels, black-owned hotels, as well as latino- and POC-owned hotels. There are U.S lists (see [4S 2019 in New Orleans](https://www.4s2019.org/accommodation/) conference for more), however for Canada we did a lot of searching based on cit
```

Comment:
Not sure about the end of this sentence: `searching based on cit` -> `searching based on city`?

Reply:
yes, should be!

Reply:
fixed locally